smoke-tests/designer/features/pageobjects/sections/createComponent.section.js

```diff
@@ -1,10 +1,14 @@
-const Page = require("./basePage");
+const Page = require("../pages/basePage");
 
-class AddComponentPage extends Page {
+class CreateComponentSection extends Page {
   get parent() {
     return browser.$(".panel--flyout");
   }
 
+  get errorSummary() {
+    return this.parent.$(".govuk-error-summary");
+  }
+
   get addComponent() {
```

Comment:
being a pedant, but worth renaming this to createComponent since you already have elsewhere

Reply:
Fixed